### DISCUSSION TITLE
Lua-Spore 504 error

### DISCUSSION THREAD

**Discussion by mendelct** [koreader/koreader#8786] [Q&A]
I'm trying to build the emulator and I'm getting this error, Cloning into 'lua-Spore'...
fatal: unable to access 'https://framagit.org/fperrad/lua-Spore/': The requested URL returned error: 504. In my browser I can open the page so I don't know what is causing this error.

**Comment by Frenzie** [comment]
Probably just a temporary fluke. (That's why some use monorepos.)

**Reply by mendelct** [reply]
A week long fluke?

**Reply by Frenzie** [reply]
Might be something to mention in the OP in the future. ;-) Which OS?

**Reply by mendelct** [reply]
Ubuntu 20, this also happens on WSL2

**Reply by Frenzie** [reply]
What does it say if you run

git clone https://framagit.org/fperrad/lua-Spore.git

**Reply by mendelct** [reply]
On the ubuntu it cloned it, and when I tried building it it worked. However on the WSL2 it gave me fatal: unable to access 'https://framagit.org/fperrad/lua-Spore.git/': server certificate verification failed. CAfile: /etc/ssl/certs/ca-certificates.crt CRLfile: none. 
BTW the Ubuntu is a VM on the same box as the WSL.

**Reply by Frenzie** [reply]
In that case it sounds like they may be fairly outdated and need an apt update && apt upgrade.

**Reply by Frenzie** [reply]
See this quick search result: https://www.ssls.com/blog/lets-encrypt-root-certificate-expiry-causes-issues-for-users/

(October 2021 is on purpose, not a mistake on my end.)

**Reply by mendelct** [reply]
The upgrade did the job.
Thanks
